tests/checkpoint/test_checkpoint.py

```diff
@@ -1028,10 +1184,10 @@ def test_newstyle_checkpoint_config_substitution_simple(
                 },
             ],
             evaluation_parameters={
-                "environment": "runtime-$GE_ENVIRONMENT",
+                "environment": "runtime-my_ge_environment",
                 "tolerance": 1.0e-2,
-                "aux_param_0": "runtime-$MY_PARAM",
-                "aux_param_1": "1 + $MY_PARAM",
+                "aux_param_0": "runtime-1",
```

Comment:
Same comment as above -- I thought that `$MY_PARAM` would work now...

Reply:
See above comment (this is in the "expected" part)